examples/xonotic/go.mod

```diff
@@ -0,0 +1,19 @@
+module agones.dev/agones/examples/xonotic
```

Comment:
Can you still run `make build` from this directory to build this locally outside of the docker container?

Reply:
Before making these changes, I am not able to build image but `go build -o wrapper .` command creating the binary locally.
<img width="1657" alt="image" src="https://user-images.githubusercontent.com/94632788/178660153-0d67abbf-84f7-4e53-a4ab-8098451bacef.png">

After making changes, I am able to build it both ways.